### PR TITLE
[841] Re-enable AuditSearchEventJob

### PR DESCRIPTION
**Pull request by davidwinter**
## Trello card URL:

https://trello.com/c/fIK1pGfH/841-reenable-auditsearcheventjob

## Changes in this PR:

* Updates the `AuditSearchEventJob` to use `AuditData` instead
* Add search event data to daily `spreadsheets:update` rake task
* Add Terraform changes for new environment variable for the relevant worksheet ID

## Next steps:

- [ ] Add a new sheet to the audit spreadsheet
- [ ] Add the GID of the new sheet to the `tfvars` files
- [ ] Deploy Terraform

### CODE REVIEW

File: app/jobs/audit_search_event_job.rb [hunk 1]
```diff
@@ -1,11 +1,7 @@
 class AuditSearchEventJob < SpreadsheetWriterJob
```
Comment (erbridge) [inline app/jobs/audit_search_event_job.rb]: Shouldn't this inherit from `ApplicationJob` not `SpreadsheetWriterJob`?

Comment (pezholio) [reply app/jobs/audit_search_event_job.rb]: Nice catch. I don't think we need `SpreadsheetWriterJob` around at all now either

Comment (davidwinter) [reply app/jobs/audit_search_event_job.rb]: Great catch again! Sorted.

**Comment by erbridge** [comment]
[Reenable AuditSearchEventJob](https://trello.com/c/fIK1pGfH/841-reenable-auditsearcheventjob)

**Comment by pezholio** [comment]
This is all rebased now, and I've added the GID to the tfvars, so might be nice to go into #888

File: config/initializers/google_drive_vars.rb [hunk 2]
```diff
@@ -6,5 +6,6 @@
   vacancy_publish_feedback: ENV['AUDIT_VACANCY_PUBLISH_FEEDBACK_WORKSHEET_GID'],
   general_feedback: ENV['AUDIT_GENERAL_FEEDBACK_WORKSHEET_GID'],
   interest_expression: ENV['AUDIT_EXPRESS_INTEREST_WORKSHEET_GID'],
-  subscription_creation: ENV['AUDIT_SUBSCRIPTION_CREATION_WORKSHEET_GID']
+  subscription_creation: ENV['AUDIT_SUBSCRIPTION_CREATION_GID'],
```
Comment (erbridge) [inline config/initializers/google_drive_vars.rb]: Merge / rebase error: This shouldn't have been renamed.

Comment (pezholio) [reply config/initializers/google_drive_vars.rb]: Gaaah. Fixed now

**Pull request merged**